src/constants/Config.ts

```diff
@@ -0,0 +1,16 @@
+export const defaultSessionParams = {
```

Comment:
Just to verify, it might be good to keep these defaults similar to minimum required namespaces we have in `@walletconnect/ethereum-provider` package.